src/ripple/app/main/Amendments.cpp

```diff
@@ -39,6 +39,11 @@ supportedAmendments ()
 {
     return
     {
+        { "4C97EBA926031A7CF7D7B36FDE3ED66DDA5421192D63DE53FFB46E43B9DC8373 MultiSign" },
+        { "C1B8D934087225F509BEB5A8EC24447854713EE447D277F69545ABFA0E0FD490 Tickets" },
+        { "DA1BD556B42D85EA9C84066D028D355B52416734D3283F85E216EA5DA6DB7E13 SusPay" },
+        { "6781F8368C4771B83E8B821D88F580202BCB4228075297B19E4FDC5233F1EFDC TrustSetAuth" },
+        { "42426C4D4F1009EE67080A9B7965B44656D7714D104A72F9B4369F97ABF044EE FeeEscalation" }
     };
```

Comment:
Are we actually deploying five live amendments that we plan to enable on the real network? I don't think fee escalation changes transaction processing, so I don't think it needs an amendment. And the others should be rolled up into one so that clients don't have to think about 16 possible combinations and so that we don't clutter the ledger.